tests/uptane_fake_http_test.cc

```diff
@@ -0,0 +1,99 @@
+#include <gtest/gtest.h>
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "fsstorage.h"
+#include "logger.h"
+#include "test_utils.h"
+#include "uptane/uptanerepository.h"
+#include "utils.h"
+
+void doDeviceRegister(const std::string &device_id, Config &conf, bool expect) {
+  system((std::string("rm -rf ") + conf.tls.certificates_directory.string() + "/*").c_str());
```

Comment:
Any particular reason not to use boost::filesystem::remove_all()? The directory in question is specified in the toml file as tests/test_uptane/, which is intended to be a purely temporary directory for the duration of the test.